### ISSUE TITLE
Platform Events in New Apex Trigger SObject list

### ISSUE THREAD

**Issue by superdave7380** [xjsender/haoide#173]
Hiya. I'd like to be able to see Platform Event sObjects in the list of selectable sObjects when I click New ApexTrigger.

Thanks very much! Keep up the stellar work!

**Comment by xjsender** [comment]
@superdave7380 , you mean ``Event`` Object, it is already available.

**Comment by superdave7380** [comment]
@xjsender , no. **Platform Events**. They're new-ish. They showed up a few releases ago. You can find them in the Setup menu. And once created, you can create After Insert (and only After Insert) triggers for them.
https://developer.salesforce.com/docs/atlas.en-us.platform_events.meta/platform_events/platform_events_intro.htm

**Comment by xjsender** [comment]
@superdave7380, you know, ``Platform Events`` was released since ``Spring 17``, which is also mean ``Platform Events`` object is accessible since ``API v39.0``, so you should change the project API version to above 39.0.

A bug is also fixed at the same time.

**Comment by superdave7380** [comment]
@xjsender I've been on v41.0 for a while now. Still, when I do **New > New ApexTrigger**, I don't see any ___e_ objects. My Platform Event, `Order_Event__e`, _is_ visible in the objects folder, i.e., I do have an `Order_Event__e.object`.

**Comment by superdave7380** [comment]
If this helps at all, know that Platform Events are _NOT_ queryable. For example, in the Workbench, you _can_ see the describe info for Platform Events, but you _can't_ see actual records.

**Comment by xjsender** [comment]
@superdave7380 , yes, ``Platform Event`` is a very specify type, only ``Platform Event `` is not queryable and triggerable. 

I need to release the new version before you can use it.

**Comment by superdave7380** [comment]
Perfect. Thanks.

**Comment by xjsender** [comment]
@superdave7380 , it's release just now, you can check at ``package control`` when it's available, after it is available, ``3.5.4`` will appear in [haoide at Package Control](https://packagecontrol.io/packages/haoide)

**Comment by superdave7380** [comment]
Awesome. Thanks very much for maintaining this plugin.

**Comment by superdave7380** [comment]
Sorry, @xjsender. `3.5.4` hasn't resolved the issue. `__e` Sobjects still aren't available in the triggers list. Not a critical issue, as I can still create the trigger file manually by right-clicking the triggers folder..

**Comment by xjsender** [comment]
You should reload your sobject cache in the cache of the main menu.发自我的iPhone------------------ Original ------------------From: David Cohen  <notifications@github.com>Date: 周二,2月 13,2018 2:32 上午To: xjsender/haoide <haoide@noreply.github.com>Cc: Hao Liu <mouse.mliu@gmail.com>, Mention <mention@noreply.github.com>Subject: Re: [xjsender/haoide] Platform Events in New Apex Trigger SObjectlist (#173)Sorry, @xjsender. 3.5.4 hasn't resolved the issue. __e Sobjects still aren't available in the triggers list. Not a critical issue, as I can still create the trigger file manually by right-clicking the triggers folder..

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/xjsender/haoide","title":"xjsender/haoide","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/xjsender/haoide"}},"updates":{"snippets":[{"icon":"PERSON","message":"@superdave7380 in #173: Sorry, @xjsender. `3.5.4` hasn't resolved the issue. `__e` Sobjects still aren't available in the triggers list. Not a critical issue, as I can still create the trigger file manually by right-clicking the triggers folder.."}],"action":{"name":"View Issue","url":"https://github.com/xjsender/haoide/issues/173#issuecomment-365017603"}}}<
 /script></div></div>
</body>
</html>

**Comment by superdave7380** [comment]
Sorry, that doesn't do it
